### PR TITLE
Update Jenkinsfile_CNP

### PR DESCRIPTION
**Pull request by scott-robertson1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-61

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -6,7 +6,7 @@ def type = "java"
 def product = "darts"
 def component = "api"
 
-def branchesToSync = ['demo']
+def branchesToSync = ['sbox']
```
Comment (jackmaloney) [inline Jenkinsfile_CNP:9]: We still want demo as well

Comment (jackmaloney) [reply Jenkinsfile_CNP:9]: Plus Dave has already done on gateay service like this https://github.com/hmcts/darts-gateway/blame/ccaf77a060baba4ae485e8d89ed957b4aad36619/Jenkinsfile_CNP#L9 
Can we keep the naming consistent so we don't create confusion?

Comment (davet1985) [reply Jenkinsfile_CNP:9]: sbox might well be right?! 🤷‍♂️

Comment (scott-robertson1) [reply Jenkinsfile_CNP:9]: I've just being going off the files in the flux repo for it, happy to change it though ?

**Pull request closed without merging**